src/logic/calculate.js

```diff
@@ -0,0 +1,29 @@
+import Operate from './operate';
+
+const Calculate = (dataObj, buttonName) => {
+  let { total, next, operation } = dataObj;
+  const operators = ['+', '÷', 'x', '-', '%'];
+
```

Comment:
It is better to name your javascript functions starting with a lower case letter. Check out some highlighted JS coding conventions [here](https://www.w3schools.com/js/js_conventions.asp). __Note:__ This applies to the `operate` function as well.